codegen/src/main/java/org/web3j/codegen/SolidityFunctionWrapper.java

```diff
@@ -122,16 +124,13 @@ void generateJavaFiles(
 
         TypeSpec.Builder classBuilder = createClassBuilder(className, bin);
 
-        classBuilder.addMethod(buildConstructor(Credentials.class, CREDENTIALS));
         classBuilder.addMethod(buildConstructor(TransactionManager.class,
                 TRANSACTION_MANAGER));
         classBuilder.addFields(buildFuncNameConstants(abi));
         classBuilder.addMethods(
                 buildFunctionDefinitions(className, classBuilder, abi));
-        classBuilder.addMethod(buildLoad(className, Credentials.class, CREDENTIALS));
```

Comment:
We should be able to provide a TransactionManager or Credentials in a generated class - this looks like it removes the credentials.

Reply:
Let me roll back this commit and get the 3.6 out, given that more work needs to be done on this.